samcli/commands/local/start_api/cli.py

```diff
@@ -47,6 +47,7 @@
     default="public",
     help="Any static assets (e.g. CSS/Javascript/HTML) files located in this directory " "will be presented at /",
 )
+@click.option("--localhost", help="help set localhost running in Docker")
```

Comment:
I don't think `localhost` is the correct term here. It has a specific meaning as the loopback address, but Docker containers can be started anywhere (as all communication happens through a socket).

Reply:
How about `container_host`? And also I'm just thinking this option can also be used in local invoke. So should we put it in the common options?

Reply:
If there's no ambiguity as to what container it applies to, then I think that's reasonable. As for common options, sure if it makes sense to be used by other subcommands.